spring-integration-file/src/test/java/org/springframework/integration/file/remote/gateway/RemoteFileOutboundGatewayTests.java

```diff
@@ -1009,214 +1011,216 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
+			return null;
+		}
 
-	@Override
-	public TestLsEntry[] list(String path) throws IOException {
-		return null;
-	}
+		@Override
+		public void read(String source, OutputStream outputStream)
+				throws IOException {
+		}
 
-	@Override
-	public void read(String source, OutputStream outputStream)
-			throws IOException {
-	}
+		@Override
+		public void write(InputStream inputStream, String destination)
+				throws IOException {
+		}
 
-	@Override
-	public void write(InputStream inputStream, String destination)
-			throws IOException {
-	}
+		@Override
+		public void append(InputStream inputStream, String destination)
+				throws IOException {
+		}
 
-	@Override
-	public void append(InputStream inputStream, String destination)
-			throws IOException {
-	}
+		@Override
+		public boolean mkdir(String directory) throws IOException {
+			return true;
+		}
 
-	@Override
-	public boolean mkdir(String directory) throws IOException {
-		return true;
-	}
+		@Override
+		public boolean rmdir(String directory) throws IOException {
+			return true;
+		}
 
-	@Override
-	public boolean rmdir(String directory) throws IOException {
-		return true;
-	}
+		@Override
+		public void rename(String pathFrom, String pathTo)
+				throws IOException {
+		}
 
-	@Override
-	public void rename(String pathFrom, String pathTo)
-			throws IOException {
-	}
+		@Override
+		public void close() {
+			open = false;
+		}
 
-	@Override
-	public void close() {
-		open = false;
-	}
+		@Override
+		public boolean isOpen() {
+			return open;
+		}
 
-	@Override
-	public boolean isOpen() {
-		return open;
-	}
+		@Override
+		public boolean exists(String path) throws IOException {
+			return true;
+		}
 
-	@Override
-	public boolean exists(String path) throws IOException {
-		return true;
-	}
+		@Override
+		public String[] listNames(String path) throws IOException {
+			return null;
+		}
 
-	@Override
-	public String[] listNames(String path) throws IOException {
-		return null;
-	}
+		@Override
+		public InputStream readRaw(String source) throws IOException {
+			return null;
+		}
 
-	@Override
-	public InputStream readRaw(String source) throws IOException {
-		return null;
-	}
+		@Override
+		public boolean finalizeRaw() throws IOException {
+			return false;
+		}
 
-	@Override
-	public boolean finalizeRaw() throws IOException {
-		return false;
-	}
+		@Override
+		public Object getClientInstance() {
+			return null;
+		}
 
-	@Override
-	public Object getClientInstance() {
-		return null;
 	}
 
-}
 
-class TestRemoteFileOutboundGateway extends AbstractRemoteFileOutboundGateway<TestLsEntry> {
 
-	@SuppressWarnings({"rawtypes", "unchecked"})
-	public TestRemoteFileOutboundGateway(SessionFactory sessionFactory,
-										 String command, String expression) {
-		super(sessionFactory, Command.toCommand(command), expression);
-		this.setBeanFactory(mock(BeanFactory.class));
-	}
+	class TestRemoteFileOutboundGateway extends AbstractRemoteFileOutboundGateway<TestLsEntry> {
```

Comment:
I am not sure why you made these inner classes, but in any case, this one should be `static`.